zilliqa/src/consensus.rs

```diff
@@ -113,12 +116,13 @@ pub struct Consensus {
     secret_key: SecretKey,
     config: NodeConfig,
     message_sender: MessageSender,
-    block_store: BlockStore,
+    pub block_store: BlockStore,
```

Comment:
if you make block store public, you can avoid having consensus having duplicated methods to access it